gozw.go

```diff
@@ -348,6 +365,9 @@ func (c *Client) AddNodeWithProgress(ctx context.Context, progress chan PairingP
 			currentProgress := node.InterviewedCommandClassCount
 
 			if currentProgress == lastReportedInterviewLength {
+				// Re-checking too quickly is impolite
```

Comment:
Good call. I didn't realize we were busy waiting here.

Reply:
Pretty sure this is what was causing a fair share of the timing weirdness